### ISSUE TITLE
Reloading the code with F5  does not reload the code as supposed to, instead just clears the screen (apparently)

### ISSUE THREAD

**Issue by FishyBusinessYT** [pokepetter/ursina#187]
Hello! 

When I press F5 to reload, it just resets the window as a whole.... There's just a single entity in the "game" _(not even an actual game, but a test file to try out functions and ideas_) I've made, **but it does not appear after reloading**. Needless to say, any attempt to interact with said entity **just crashes the window**. 

This is the log:
[Ursina_not_reloading_code_log.txt](https://github.com/pokepetter/ursina/files/6410283/Ursina_not_reloading_code_log.txt)

This is my code file (some of it commented out, should be considered as irrelevant, since it was run just that way):
[main.txt](https://github.com/pokepetter/ursina/files/6410286/main.txt)

**Comment by baluR220** [comment]
Hi!

**TL;DR**
This problem is caused by line 15 in your code
`app.setBackgroundColor(color.white)`
set bg color with this command:
`window.color = color.white`

**More details**
When you want ursina to reload your code, it parses source code and throws away some lines including lines like
`app = Ursina()`
then exec() is called. 
i suppose otherway all application should be closed in order to create new instance of Ursina() class which is meaningless in scope of reloading code withoit closing application.
so, in your case exception is thrown on exec() when there is no definition of the app instance in scope of 'new' source code.

**Comment by FishyBusinessYT** [comment]
Thanl you! I'll try fix it and update or close the issue accordingly :)
